src/cupbearer/data/transforms.py

```diff
@@ -16,13 +18,13 @@ class AdaptedTransform(Transform, ABC):
     """Adapt a transform designed to work on inputs to work on img, label pairs."""
 
     @abstractmethod
-    def __img_call__(self, img):
+    def __img_call__(self, img: torch.Tensor) -> torch.Tensor:
         pass
 
-    def __rest_call__(self, *rest):
+    def __rest_call__(self, *rest: tuple[Any, ...]) -> tuple[Any, ...]:
```

Comment:
```suggestion
    def __rest_call__(self, *rest: Any) -> tuple[Any, ...]:
```
Type annotations to `*args` apply to every arg individually, so there shouldn't be a `tuple` annotation unless each individual argument was a tuple (which I don't think is the case here?)

Reply:
Oh, makes sense.